src/containers/user/signin/Signin.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Facebook from '../../../views/social/Facebook';
+import Google from '../../../views/social/Google';
+import '../../../assets/css/style.scss';
+
+export class Signin extends Component {
+  render() {
+    const isError = this.props.error;
```

Comment:
@NiyoEric would you mind adding props validation?

Reply:
@victorkarangwa4 Thanks for pointing the line. Since we 've decided to use react-toastify, we don't need it.